packages/kit/src/runtime/server/validate-headers.js

```diff
@@ -0,0 +1,62 @@
+/** @type {Set<string>} */
+const VALID_CACHE_CONTROL_DIRECTIVES = new Set([
+	'max-age',
+	'public',
+	'private',
+	'no-cache',
+	'no-store',
+	'must-revalidate',
+	'proxy-revalidate',
+	's-maxage',
+	'immutable',
+	'stale-while-revalidate',
+	'stale-if-error',
+	'no-transform',
+	'only-if-cached',
+	'max-stale',
+	'min-fresh'
+]);
+
+const CONTENT_TYPE_PATTERN =
+	/^(application|audio|font|image|model|text|video|x-[a-z]+)\/[-+.\w]+$/i;
```

Comment:
```suggestion
	/^(application|audio|font|image|model|text|video|x-[a-z]+)\/[-+.\w]+$/i;
```

[The IANA spec](https://www.iana.org/assignments/media-types/media-types.xhtml) defines the following top-level types, some of which are not included here:
- [application](https://www.iana.org/assignments/media-types/media-types.xhtml#application)
- [audio](https://www.iana.org/assignments/media-types/media-types.xhtml#audio)
- [example](https://www.iana.org/assignments/media-types/media-types.xhtml#example)
- [font](https://www.iana.org/assignments/media-types/media-types.xhtml#font)
- [haptics](https://www.iana.org/assignments/media-types/media-types.xhtml#haptics)
- [image](https://www.iana.org/assignments/media-types/media-types.xhtml#image)
- [message](https://www.iana.org/assignments/media-types/media-types.xhtml#message)
- [model](https://www.iana.org/assignments/media-types/media-types.xhtml#model)
- [multipart](https://www.iana.org/assignments/media-types/media-types.xhtml#multipart)
- [text](https://www.iana.org/assignments/media-types/media-types.xhtml#text)
- [video](https://www.iana.org/assignments/media-types/media-types.xhtml#video)

Reply:
Thanks! Updated to reflect this.